Cannot compile readin_hdf5

Hi there,
I'm trying to build the project (using gcc 13.1.1 on archlinux) and I get this error

```bash
[ 22%] Building Fortran object CMakeFiles/hoprlibF90.dir/src/readin/readin_hdf5.f90.o
[MY_PATH]/hopr/src/readin/readin_hdf5.f90:104:20:

  104 | CHECKSAFEINT(HSize(2))
      |                    1
Error: Missing kind-parameter at (1)
[MY_PATH]/hopr/src/readin/readin_hdf5.f90:619:20:

  619 | CHECKSAFEINT(HSize(1))
      |                    1
Error: Missing kind-parameter at (1)
make[2]: *** [CMakeFiles/hoprlibF90.dir/build.make:569: CMakeFiles/hoprlibF90.dir/src/readin/readin_hdf5.f90.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:156: CMakeFiles/hoprlibF90.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

Configuring works well though, did `CC=gcc FC=gfortran ccmake ../ ` and configured as-is, Am I doing something wrong here?
Thanks for the help

--- a/src/hopr.h
+++ b/src/hopr.h
@@ -45,8 +45,8 @@
 #define SIZEOF_F(x) STORAGE_SIZE(x)/8
 
 #ifdef GNU
-#define CHECKSAFEINT(x,k)  IF(x>HUGE(1_  k).OR.x<-HUGE(1_  k))       CALL ABORT(__STAMP__,'Integer conversion failed: out of range!')
-#define CHECKSAFEREAL(x,k) IF(x>HUGE(1._ k).OR.x<-HUGE(1._ k))       CALL ABORT(__STAMP__,'Real conversion failed: out of range!')
+#define CHECKSAFEINT(x,k)  IF(x>HUGE(INT( 1,KIND=k)).OR.x<-HUGE(INT( 1,KIND=k))) CALL Abort(__STAMP__,'Integer conversion failed: out of range!')
+#define CHECKSAFEREAL(x,k) IF(x>HUGE(REAL(1,KIND=k)).OR.x<-HUGE(REAL(1,KIND=k))) CALL Abort(__STAMP__,'Real conversion failed: out of range!')
 #else
 #define CHECKSAFEINT(x,k)  IF(x>HUGE(1_  ## k).OR.x<-HUGE(1_  ## k)) CALL ABORT(__STAMP__,'Integer conversion failed: out of range!')
 #define CHECKSAFEREAL(x,k) IF(x>HUGE(1._ ## k).OR.x<-HUGE(1._ ## k)) CALL ABORT(__STAMP__,'Real conversion failed: out of range!')
@@ -86,4 +86,3 @@
 #if(PP_CGNS_INT==32)
 #  define PP_CGNS_INT_TYPE INTEGER(CGSIZE_T)
 #endif
-

